scripts/projector.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# failfast
+set -e
+
+# Define these 2 optional variables if you want to customize the default installation,
+# then copy-paste the remainder of the script:
+# - the name of the projector config (defaults to 'Idea')
+# projector_config_name=Idea
+# - the name of the IDE to install (defaults to "IntelliJ IDEA Community Edition 2020.3.4")
+# Note: to get list of available IDEs, launch `projector find`
+# projector_ide_name="IntelliJ IDEA Community Edition 2020.3.4"
+
+projector_config_name=${projector_config_name:-Idea}
+projector_ide_name=${projector_ide_name:-IntelliJ IDEA Community Edition 2020.3.4}  # Latest projector-tested IDE
+
+# Install projector-installer
+# https://github.com/JetBrains/projector-installer
+apt-get update
+apt-get install -y --no-install-recommends python3 python3-pip python3-cryptography less libxext6 libxrender1 libxtst6 libfreetype6 libxi6
+pip3 install projector-installer
+
+# Setup projector
+sudo -u ubuntu projector --accept-license ide autoinstall --config-name "${projector_config_name}" --ide-name "${projector_ide_name}" --hostname {{ .STRIGO_RESOURCE_DNS }}
+# Cannot use password as it needs to be in the URL as query parameter 'token'
+# and it cannot be different for each session in Strigo web interfaces
+# cat << EOF >> "~ubuntu/.projector/configs/${projector_config_name}/config.ini"
+# [PASSWORDS]
+# password = {{ .STRIGO_WORKSPACE_ID }}
+# ro_password = {{ .STRIGO_WORKSPACE_ID }}
+# EOF
+# sudo -u ubuntu projector --accept-license config rebuild "${projector_config_name}"
```

Comment:
should these commented-out lines be deleted? I don't understand why they are here since lines 25-26 seem to tell that one should not create this authentication approach.

Reply:
Did add the possibility to setup the password now.